### DISCUSSION TITLE
Burner Phones numbers

### DISCUSSION THREAD

**Discussion by AedenThomas** [simple-login/app#462] [Feature Requests]
Can SimpleLogin also bring a temporary number to receive OTPs?

**Comment by nguyenkims** [comment]
This is part of our long term plan. We are actually studying how to make a temporary phone service and hope to have it available at the beginning of next year. It'll probably be available in France or Europe first before being extended to other countries.

**Reply by c0nfigurati0n** [reply]
Yes! Can't wait to finally get a good aliasing service for phone numbers in the eu!

**Reply by nguyenkims** [reply]
The received SMS would probably be displayed in the app as sending them via email or SMS redirection isn't quite practical.

**Comment by sghdesign** [comment]
Just joined and love the service! Any update on this feature?!

**Reply by nguyenkims** [reply]
We are actually working on this feature so please stay tuned!

**Reply by sleeping10** [reply]
Now when Firefox Relay has rolled out there version. I have a question will this be like the Firefox Relay version that will only support one region from the start, Firefox Relay currently only supports the US and Canada?

**Comment by krestaino** [comment]
Is this feature still planned?

**Reply by ronbillock** [reply]
I am also very interested in this service. Any update?

**Reply by gregbtm** [reply]
Any update? @nguyenkims

**Comment by Asherathe** [comment]
Single-use credit card numbers and phone numbers were a big part of my interest in SimpleLogin, but years later, they still haven't been implemented. It seems like joining Proton completely threw SL off-track as a privacy service.
